expansion/src/main/java/no/unit/nva/expansion/model/ExpandedDoiRequest.java

```diff
@@ -48,11 +48,13 @@ public static ExpandedDoiRequest createEntry(DoiRequest doiRequest,
                                                  ResourceService resourceService,
                                                  TicketService ticketService)
         throws NotFoundException {
-        var expandedDoiRequest = ExpandedDoiRequest.fromDoiRequest(doiRequest, resourceService);
+        var expandedDoiRequest = ExpandedDoiRequest.fromDoiRequest(doiRequest, resourceService, expansionService);
         expandedDoiRequest.setOrganizationIds(fetchOrganizationIdsForViewingScope(doiRequest, expansionService));
         expandedDoiRequest.setMessages(expandMessages(doiRequest.fetchMessages(ticketService), expansionService));
         expandedDoiRequest.setOwner(expansionService.expandPerson(doiRequest.getOwner()));
-        expandedDoiRequest.setAssignee(expandAssignee(doiRequest, expansionService));
+        expandedDoiRequest.setAssignee(extractPersonUsername(doiRequest.getAssignee(), expansionService));
```

Comment:
maybe rename method to expandPerson? because you are not only extracting username, but also expanding it.